src/AddTodoForm.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react'; 
+
+function AddTodoForm(props) {
+    const [todoTitle, setTodoTitle] = useState('');
+
+  function handleAddTodo(event){
+    event.preventDefault();
+    console.log(todoTitle);
+    props.onAddTodo(todoTitle);
+    event.target.reset();
+  }
+  return (
+    <form onSubmit={(e) => handleAddTodo(e)}>
+      <label htmlFor="todoTitle">Title</label>
+      <input type="text" name="title" onInput={(e) => setTodoTitle(e.target.value)} />
```

Comment:
Very good of you to build a controlled form! Although this is not in the instructions for 1.3, taking this approach gives you a lot more flexibility when it comes to submission. Your use of `onInput` is correct but `onChange` is more common. Read more here in the [dev docs](https://beta.reactjs.org/reference/react-dom/components/select#props)

Reply:
Roy, thank you - I really appreciate your time and feedback. Was looking forward all day to reading your review. And I will go through all your comments and follow your recommendations.